sonar-php-plugin/src/test/java/org/sonar/plugins/php/PHPProfileDefinitionTest.java

```diff
@@ -44,7 +44,7 @@ public void should_create_sonar_way_profile() {
     BuiltInQualityProfilesDefinition.BuiltInQualityProfile profile = context.profile("php", "Sonar way");
     assertThat(profile.language()).isEqualTo(Php.KEY);
     assertThat(profile.name()).isEqualTo(PHPProfileDefinition.SONAR_WAY_PROFILE);
-    assertThat(profile.rules().size()).isGreaterThan(50);
+    assertThat(profile.rules().size()).isGreaterThan(51);
```

Comment:
I don't think this change is required. It's intended to test that profile is not at all empty